aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -706,6 +707,10 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         return inputConnectionRef?.get()!!
     }
 
+    open fun getPreformatBackgroundAlpha(styles: TypedArray): Float {
```

Comment:
Let's add a quick comment here on why we're exposing this method.

Reply:
Done. I have added a proper comment. 
It's ready for another round of review.